### PR TITLE
Add Georgian government holiday for May 17, 2024

### LINKED ISSUE #1795: May 17th, 2024 is a holiday in Georgia
https://georgiatoday.ge/on-may-17-to-be-a-holiday-in-georgia/

### PR DESCRIPTION
**Pull request by PPsyrius**
## Proposed change

Add Georgian `government` holiday for May 17, 2024 "Day of Family Sanctity and Respect for Parents" (ოჯახის სიწმინდისა და მშობლების პატივისცემის დღე) as decreed by [Resolution No. 167](https://www.matsne.gov.ge/ka/document/view/6173967). Ukrainian translation is cross-referenced with [[1]](https://pravlife.org/uk/content/gruzynska-cerkva-vidznachyla-den-zgurtovanosti-ta-svyatosti-simyi) .

Resolves #1795.

## Type of change

- [ ] New country/market holidays support (thank you!)
- [x] Supported country/market holidays update (calendar discrepancy fix, localization)
- [ ] Existing code/documentation/test/process quality improvement (best practice, cleanup, refactoring, optimization)
- [ ] Dependency update (version deprecation/upgrade)
- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] Breaking change (a code change causing existing functionality to break)
- [ ] New feature (new python-holidays functionality in general)

## Checklist

- [x] I've followed the [contributing guidelines][contributing-guidelines]
- [x] I've run `make pre-commit`, it didn't generate any changes
- [x] I've run `make test`, all tests passed locally

[contributing-guidelines]: https://github.com/vacanza/python-holidays/blob/dev/CONTRIBUTING.rst
[docs]: https://github.com/vacanza/python-holidays/tree/dev/docs/source

### CODE REVIEW

File: holidays/locale/uk/LC_MESSAGES/GE.po [hunk 1]
```diff
@@ -90,3 +90,7 @@ msgstr "Свято Светіцховлоба, Ризи Господньої"
 #. Saint George's Day.
 msgid "გიორგობა"
 msgstr "День святого Георгія"
+
+#. Day of Family Purity and Respect for Parents.
+msgid "ოჯახის სიწმინდისა და მშობლების პატივისცემის დღე"
+msgstr "День згуртованості та святості сім'ї"
```
Comment (arkid15r) [inline holidays/locale/uk/LC_MESSAGES/GE.po:96]:
```suggestion
msgstr "День святості родини та поваги до батьків"
```

File: snapshots/countries/GE_COMMON.json [hunk 2]
```diff
@@ -569,6 +569,7 @@
     "2024-05-06": "Easter Monday",
     "2024-05-09": "Day of Victory over Fascism",
     "2024-05-12": "Saint Andrew's Day",
+    "2024-05-17": "Day of Family Purity and Respect for Parents",
```
Comment (arkid15r) [inline snapshots/countries/GE_COMMON.json:572]:
```suggestion
    "2024-05-17": "Day of Family Sanctity and Respect for Parents",
```

File: holidays/countries/georgia.py [hunk 3]
```diff
@@ -96,3 +100,15 @@ class GE(Georgia):
...
+
+
+class GeorgiaStaticHolidays:
+    """
+    References:
+        - https://www.matsne.gov.ge/ka/document/view/6173967
+    """
+
+    special_government_holidays = {
+        # Day of Family Purity and Respect for Parents.
```
Comment (arkid15r) [inline holidays/countries/georgia.py:112]:
```suggestion
        # Day of Family Sanctity and Respect for Parents.
```

File: holidays/locale/en_US/LC_MESSAGES/GE.po [hunk 4]
```diff
@@ -90,3 +90,7 @@ msgstr "Holiday of Svetitskhovloba, Robe of Jesus"
 #. Saint George's Day.
 msgid "გიორგობა"
 msgstr "Saint George's Day"
+
+#. Day of Family Purity and Respect for Parents.
```
Comment (arkid15r) [inline holidays/locale/en_US/LC_MESSAGES/GE.po:94]:
```suggestion
#. Day of Family Sanctity and Respect for Parents.
```

File: holidays/locale/ka/LC_MESSAGES/GE.po [hunk 5]
```diff
@@ -90,3 +90,7 @@ msgstr ""
 #. Saint George's Day.
 msgid "გიორგობა"
 msgstr ""
+
+#. Day of Family Purity and Respect for Parents.
```
Comment (arkid15r) [inline holidays/locale/ka/LC_MESSAGES/GE.po:94]:
```suggestion
#. Day of Family Sanctity and Respect for Parents.
```

File: holidays/locale/uk/LC_MESSAGES/GE.po [hunk 6]
```diff
@@ -90,3 +90,7 @@ msgstr "Свято Светіцховлоба, Ризи Господньої"
 #. Saint George's Day.
 msgid "გიორგობა"
 msgstr "День святого Георгія"
+
+#. Day of Family Purity and Respect for Parents.
```
Comment (arkid15r) [inline holidays/locale/uk/LC_MESSAGES/GE.po:94]:
```suggestion
#. Day of Family Sanctity and Respect for Parents.
```

File: holidays/locale/en_US/LC_MESSAGES/GE.po [hunk 7]
```diff
@@ -90,3 +90,7 @@ msgstr "Holiday of Svetitskhovloba, Robe of Jesus"
...
+msgid "ოჯახის სიწმინდისა და მშობლების პატივისცემის დღე"
+msgstr "Day of Family Purity and Respect for Parents"
```
Comment (arkid15r) [inline holidays/locale/en_US/LC_MESSAGES/GE.po:96]:
```suggestion
msgstr "Day of Family Sanctity and Respect for Parents"
```

File: tests/countries/test_georgia.py [hunk 8]
```diff
@@ -27,6 +28,12 @@ def test_country_aliases(self):
     def test_no_holidays(self):
         self.assertNoHolidays(Georgia(years=1990))
 
+    def test_family_purity_holiday(self):
```
Comment (arkid15r) [inline tests/countries/test_georgia.py:31]:
```suggestion
    def test_family_sanctity_day(self):
```

File: holidays/locale/uk/LC_MESSAGES/GE.po (hunk 1 above)
Comment (KJhellico) [reply holidays/locale/uk/LC_MESSAGES/GE.po:96]: Yes, it's a more accurate translation.

**Pull request merged**